sources/packages/api/src-sql/User/Create-User.sql

```diff
@@ -0,0 +1,26 @@
+-- Users Table
+CREATE TABLE IF NOT EXISTS users ( 
+  id SERIAL PRIMARY KEY,
+  user_name VARCHAR(300) NOT NULL UNIQUE,
+  email VARCHAR(300) NOT NULL,
```

Comment:
should we name it as 'verified_email_address'? just so that it is clear that we never are going to store unverified email addresses
if not, then lets atleast name it as 'email_address' instead of 'email'

Reply:
Main data is stored in KC, storing this data as placeholder for now, in future we will revisit.  And what is the difference between email and email_address?

Reply:
two different nouns. Calling email address as 'email' really is informal language. Former is the content and latter is destination